### PR TITLE
implement api call ExportEntry

### PR DESCRIPTION
**Pull request by ovv**

### CODE REVIEW

File: entries.go [hunk 1]
```diff
@@ -218,3 +218,13 @@ func GetEntry(bodyByteGetterFunc BodyByteGetter, articleID int) (Item, error) {
 	err = json.Unmarshal(body, &item)
 	return item, err
 }
+
+// ExportEntry queries the API for a specific article content according to the API request /entries/ID
+func ExportEntry(bodyByteGetterFunc BodyByteGetter, articleID int, format string) ([]byte, error) {
+	entryURL := Config.WallabagURL + "/api/entries/" + strconv.Itoa(articleID) + "/export." + format
```
Comment (Strubbl) [inline entries.go:224]: I am unsure wether we should check the `format` string for a valid value. I mean, that it is a string, which the API can handle (as of now xml|json|txt|csv|pdf|epub|mobi according to https://app.wallabag.it/api/doc#get--api-entries-{entry}-export.{_format}). Or shall we let the wallabag server handle invalid format strings?

File: entries.go [hunk 2]
```diff
@@ -218,3 +218,13 @@ func GetEntry(bodyByteGetterFunc BodyByteGetter, articleID int) (Item, error) {
 	err = json.Unmarshal(body, &item)
 	return item, err
 }
+
+// ExportEntry queries the API for a specific article content according to the API request /entries/ID
```
Comment (Strubbl) [inline entries.go]: I would suggest:
`// ExportEntry queries the API to retrieve a single entry in a predefined format according to the API request /entries/ID/export.FORMAT`

**Review by Strubbl — CHANGES_REQUESTED** [review]
Do you have an idea how to write a test for this func?

File: entries.go (hunk 1 above)
Comment (ovv) [reply entries.go:224]: I don't think we should. In both case if the `format` is invalid an error will be returned. If we hardcode the formats on the client and a future wallabag version implement new formats the client would require an update, we should avoid that.

**Comment by ovv** [comment]
Regarding the tests:
There isn't much to test for this as we don't do anything with the request body. The test would only check the behaviour of `bodyByteGetterFunc`, but this function is already tested elsewhere.

**Review by Strubbl — APPROVED** [review]
Thanks a lot for your feedback and your PRs

**Pull request merged**